core/templates/pages/collection-editor-page/settings-tab/collection-settings-tab.component.ts

```diff
@@ -13,21 +13,20 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the history tab of the collection editor.
+ * @fileoverview Component for the settings tab of the collection editor.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
+import { Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-angular.module('oppia').directive('collectionHistoryTab', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
-    return {
-      restrict: 'E',
-      scope: {},
-      bindToController: {},
-      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/pages/collection-editor-page/history-tab/' +
-        'collection-history-tab.directive.html'),
-      controllerAs: '$ctrl',
-      controller: [function() {}]
-    };
-  }]);
+@Component({
+  selector: 'collection-settings-tab',
+  templateUrl: './collection-settings-tab.component.html',
+  styleUrls: []
+})
```

Comment:
Why is git showing this file (settings tab) to be a renamed history-tab? git mv screw up?

Reply:
Yeah, it worked for all the others, but I have no idea why it didn't here. Even showed create/delete locally